charts/flyte-core/values-eks.yaml

```diff
@@ -111,6 +111,7 @@ flytepropeller:
       ephemeral-storage: 1Gi
       memory: 2Gi
   cacheSizeMbs: 1024
+  client_secret: foobar
```

Comment:
We only need to set it in values file, right? why do we also need to do that in vlaues-eks?

Reply:
yes